src/main/java/com/neva/felix/webconsole/plugins/search/core/OsgiExplorer.java

```diff
@@ -61,28 +62,50 @@ public OsgiExplorer(BundleContext context) {
 	}
 
 	public File findDir(Long bundleId) {
-		return new File(String.format(BUNDLE_PATH_FORMAT, context.getProperty(BUNDLE_STORAGE_PROP), bundleId));
+		String bundleStorage = context.getProperty(BUNDLE_STORAGE_PROP);
```

Comment:
Leading slash is just wrong configuration. On what type of osgi container you noticed problems? Maybe I can reproduce problem. I tested plugin on both windows and ubuntu. Maybe leading dot and slash need to be handled separately.. I will look at this. About PR . I don't want to put all of this logging entries. In my opinion they are messy...